spec/models/entity_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe Group, type: :model do
+  before(:all) do
+    @user = User.first
+    @user ||= User.create(name: 'Fatema', email: 'fatemanazari78@gmail.com', password: 'Monis_7813')
+
+    @group = Group.first
+    @group ||= Group.create(name: 'Grouptest', icon: 'grouptesicon', author: @user)
+
+    @entity = Entity.first
+    @entity ||= Entity.create(name: 'Entitytest', amount: 10, group: @group, author: @user)
+  end
+
+  it 'Entity should be valid with a name and amount' do
+    expect(@entity).to be_valid
+  end
+
+  it 'Entity name cannot be empty' do
+    @entity.name = ''
+    expect(@entity).to_not be_valid
+  end
+
+  it 'Entity amount should be greater than or equal to 1' do
+    @entity.amount = 0
```

Comment:
- [ ] As suggested earlier, 6 out of the tests written for your project model are failing 👇, kindly resolve it. 

<img width="452" alt="codereview" src="https://github.com/78fatemanazari/ezPay-rails-app/assets/103386183/f6f9a41c-0c59-4c7b-bc33-3adcf31342f4">